packages/admin/cms-admin/src/pages/pageTree/PageTreeContext.ts

```diff
@@ -11,6 +11,7 @@ export interface PageTreeContext {
     allCategories: AllCategories;
     currentCategory: string;
     documentTypes: Record<DocumentType, DocumentInterface>;
+    getDocumentTypesByCategory?: (category: string) => Record<DocumentType, DocumentInterface>;
```

Comment:
Why not change the documentTypes in the context as well instead of adding this new function?